tuning/automation/PerformanceAnalysis.py

```diff
@@ -102,6 +102,8 @@ def fillCallCounts(problemMapper, callCounts, callCountNN, callCountNNstrided, c
             midList = list()
```

Comment:
problemMapper is a list of dictionaries so i don't think you need to iterate over it here:

see if this works

   for problem in problemMapper:
        midList = list()
        midList.append(problem["f"])
        midList.append(problem["transposeA"])
        midList.append(problem["transposeB"])
        theCount = problem["i"]
        if theCount == 10:
            theCount = 1
        callCounts.append(midList)

Reply:
My offsets did not translate in the last comment

Reply:
Hmmm, I get the following error:

  File "../Tensile/tuning/automation/PerformanceAnalysis.py", line 102, in fillCallCounts
    midList.append(problem["f"])
TypeError: list indices must be integers or slices, not str

Reply:
OK. This means I was wrong. The problem mapper is then a list.